bin/upload_buildkite_test_analytics_junit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+FILE_PATH=$1
+ANALYTICS_TOKEN=$2
+
+if [ -z "$FILE_PATH" ]; then
+	echo "You must pass a path to the file you want to upload to Test Analytics"
+	exit 1
+fi
+
+if [ -z "$ANALYTICS_TOKEN" ]; then
+	echo "You must pass a Buildkite Analytics Token"
+	exit 1
+fi
```

Comment:
I don't think these `-z` checks will run, because of the `-u` in the shebang:

<img width="1195" alt="image" src="https://user-images.githubusercontent.com/1218433/181128777-fcc1cf3b-5a64-4d2e-aec6-6e7abcb51805.png">

Reply:
Ah yep – addressed in https://github.com/Automattic/bash-cache-buildkite-plugin/pull/29/commits/cc0975357ce1036cd31cd76a263ba3e4df55bf1e